src/main/java/com/kenshoo/xlsx2csv/XlsxToCsvConverter.java

```diff
@@ -86,15 +89,24 @@ private boolean isLastLine(boolean hasNext) {
         return !hasNext;
     }
 
-    private Iterator<Row> getRowIterator(InputStream inputStream) {
+    private Optional<Iterator<Row>> getRowIterator(InputStream inputStream) {
         final Workbook workbook = StreamingReader.builder()
                 .rowCacheSize(10)
                 .bufferSize(1024)
```

Comment:
Why did we choose these values? Should they be controlled by configuration? Optional parameters in the API call?